### PR TITLE
Th/query monitor agent 2

### PR DESCRIPTION
**Pull request by thomashoffman**
Monitor agent changes.

### CODE REVIEW

File: src/main/java/com/prevoty/PrevotyClient.java [hunk 1]
```diff
@@ -761,6 +787,60 @@ public QueryResult AnalyzeQuery(String query, String configurationKey) throws Ne
     }
 
     /**
+     * Analyze multiple queries for validity and secure
```
Comment (michaelcrampon) [inline src/main/java/com/prevoty/PrevotyClient.java]: and secure?

File: src/main/java/com/prevoty/QueryIntelligence.java [hunk 2]
```diff
@@ -1,18 +1,38 @@
...
-    public Set<QueryColumnObject> columnUpdates;
-    public Set<QueryTableObject> rowDeletes;
-    public boolean hasAdmin;
-    public Set<QueryFunctionCall> functionCalls;
+public class QueryIntelligence {
+    public String statement_type;
+    public String sanitized_statement;
+    public int hashed_statement;
+    public boolean has_comment;
+    public int comparisons;
```
Comment (michaelcrampon) [inline src/main/java/com/prevoty/QueryIntelligence.java]: comparisons has been updated to be a hash map of string:bools

**Comment by michaelcrampon** [comment]
Looks good.

**Comment by jrozner** [comment]
Everything looks reasonable here

**Pull request merged**